tests/integration/test_manifest_generation.py

```diff
@@ -498,6 +498,670 @@ def test_single_manifest_generation_google_sheet(
         workbook.save(
             os.path.join(
                 manual_test_verification_path,
-                "TestManifestGeneration_test_single_manifest_generation_google_sheet.xlsx",
+                "TestManifestGeneration_test_single_manifest_generation_google_sheet_with_annotations.xlsx",
+            )
+        )
+
+    def test_single_manifest_generation_google_sheet_no_annotations(
+        self, manual_test_verification_path: str
+    ) -> None:
+        """
+        Download a manifest without annotations from the Schematic API and verify that it is a valid Google
+        Sheet. We are validating the following:
+
+        - The first row of the Google Sheet contains the column headers
+        - The first row is locked on scroll
+        - Each cell A-F in the first row has a comment "TBD"
+        - Cell G in the first row does not have a comment
+        - The cell C2 in "Sheet1" has a dropdown list with values from "Sheet2!C2:C5"
+        - The cell E2 in "Sheet1" has a dropdown list with values from "Sheet2!E2:E5"
```

Comment:
I also need to update my docstring, but if we operate under the assumption the order may change, these comments are outdated.